### PR TITLE
Security/General Fixes

### PR DESCRIPTION
**Pull request by theosirian**
## Security:
- [x] Update Dio Client Setup
   - Remove TLS self-signed test code
   - Timeout for connect/send/receive
   - Cancel request with response size over 4MB
- [x] Change SecureStorage iOS accessibility option to passcode 
- [x] Change bip39 dep to fix/26/uniform-random
- [x] Disallow Android Backups on AndroidManifest.xml
- [x] Disallow third-party iOS keyboards
- [x] Lock application behind phones' biometrics
- [x] Hide sensitive information on application switcher
- [x] Verify user saved their mnemonic
- [x] Validate mnemonic word count generated by the app

## General:
- [x] Remove old merge files
- [x] Update Dart SDK Version

## Screenshots
Mnemonic Save Confirmation
Locked App

### CODE REVIEW

File: ios/Runner/AppDelegate.swift [hunk 1]
```diff
@@ -10,4 +10,11 @@ import Flutter
     GeneratedPluginRegistrant.register(with: self)
     return super.application(application, didFinishLaunchingWithOptions: launchOptions)
   }
+
+  override func application(
+    _ application: UIApplication,
+    shouldAllowExtensionPointIdentifier extensionPointIdentifier: UIApplicationExtensionPointIdentifier
```
Comment (sbihel) [inline ios/Runner/AppDelegate.swift:16]:
```suggestion
    shouldAllowExtensionPointIdentifier extensionPointIdentifier: UIApplication.ExtensionPointIdentifier
```

**Review by sbihel — APPROVED** [review]
I only compiled the project for iOS.

**Comment by bumblefudge** [comment]
GTG? @theosirian wanna review before merging?

**Comment by sbihel** [comment]
Looks fine in the simulator. Didn't scan any QR code but I suppose it wasn't really the point of this PR. The CI is simply failing because of DIDKit (https://github.com/spruceid/didkit/pull/246). I think it can be merged.

**Comment by sbihel** [comment]
Known issue for the CI.

**Pull request merged**